### PR TITLE
Added new option loadAllFonts

### PR DESCRIPTION
**Pull request by ramghaju**
We have added new optional 'loadAllFonts' option in Monotype module to fix issue #318. By default, we're passing only used project fonts to webfontloader. We're adding it as new option in monotype module to load all project fonts.

In modules.rb, we're removing new lines as we're getting error message ("compiler is waiting for input")
while compiling due to new lines used in 'src/closure.js' in windows environment.

### CODE REVIEW

File: src/modules/monotype.js [hunk 1]
```diff
@@ -54,7 +55,9 @@ goog.scope(function() {
     var self = this;
     var projectId = self.configuration_['projectId'];
     var version = self.configuration_['version'];
-
+    //Expose 'mti_loadAllFonts' variable to load all fonts while using loader without webfontconfig
+    window['mti_loadAllFonts']=this.configuration_['loadAllFonts'];
```
Comment (bramstein) [inline src/modules/monotype.js]: Could you put a space before and after `=`?

Comment (bramstein) [inline src/modules/monotype.js]: Also, is exporting a global variable the right solution here?

File: README.md [hunk 2]
```diff
@@ -280,12 +280,13 @@ When using [Fonts.com web fonts](http://www.fonts.com/web-fonts/) specify your P
 WebFontConfig = {
   monotype: {
     projectId: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx',
-    version: 12345 // (optional, flushes the CDN cache)
+    version: 12345, // (optional, flushes the CDN cache)
+    loadAllFonts:true //(optional, loads all project fonts)
```
Comment (bramstein) [inline README.md]: Space after `:`.

File: src/modules/monotype.js (hunk 1 above)
Comment (ramghaju) [inline src/modules/monotype.js]: With webfontconfig setup, it is easy to get configuration without making any changes on webfontloader code. But with webfont.load method call, it is difficult to access this flag. As we're using this flag to decide whether to pass all fonts or only used fonts, we need to access this config.

We have also tried adding new function to pass configuration from loader to our publish code. But it was not working properly. So, we decided to exporting global variable.

**Comment by janaagaard75** [comment]
Any update on this pull request?

**Comment by bramstein** [comment]
I'm not totally happy with the module writing global variables. I think that needs to be addressed before we merge the change.

**Comment by ramghaju** [comment]
@bramstein Thanks for update. We will try to avoid exposing global variable. 
@janaagaard75, Will you please try using "webfontconfig " instead of "webfont.load" for now? Please check this page, it is replica of your test page using "webfontconfig".

http://run.plnkr.co/plunks/6z78HqnuAaRhqBbLe1Wd/

**Comment by bramstein** [comment]
@ramghaju Thanks. Let me know if you need any help.

**Comment by janaagaard75** [comment]
Thanks for the suggestion, @ramghaju.

Using the WebFontConfig approach might result in a flash of unstyled text, and it's really important for us to avoid this. That's one of the reasons we use the API's load method, and it works nicely for loadings fonts from Google and Typekit. So we are still looking forward for support for loadAllFonts in the API.

**Comment by ramghaju** [comment]
@bramstein, will you please have a look into my latest changes?

**Comment by janaagaard75** [comment]
Is there anything I can do to help with this pull request?

**Comment by ramghaju** [comment]
@bramstein,  any update on this?

**Pull request merged**